app/controllers/admin/trade_stores_controller.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module Admin
+  class TradeStoresController < Spree::Admin::ResourceController
+    def index
+      @stores =
+        ::Store.page(params[:page]).per(params[:per_page] || Spree::Config[:orders_per_page])
+    end
+
+    def new
+      @store = ::Store.new
+    end
+
+    def create
+      @store = ::Store.new(store_params)
+      if @store.save
```

Comment:
I'm a nut but put a line break between `@store = ...` and `if @store.save` please

Reply:
heh, its fine